### ISSUE TITLE
git error: sha1 mismatch

### ISSUE THREAD

**Issue by pfcarrier** [termux/termux-packages#1065]
Good day,

After the update to git 2.13.1-arm I cannot clone repos or run git against an existing local git repos.

Example :
```
# Attempt at cloning a public repos
$ git clone git@github.com:Liaison-Intl/upgrade_toolbelt.git
Cloning into 'upgrade_toolbelt'...
remote: Counting objects: 313, done.
remote: Total 313 (delta 0), reused 0 (delta 0), pack-reused 313
Receiving objects: 100% (313/313), 57.22 KiB | 0 bytes/s, done.
fatal: pack is corrupted (SHA1 mismatch)
fatal: index-pack failed

$ git --version
git version 2.13.1

$ apt-cache show git
Package: git
Architecture: arm
Installed-Size: 14060
Maintainer: Fredrik Fornwall @fornwall
Version: 2.13.1
Description: Distributed version control system designed to handle everything from small to very large projects with speed and efficiency
Description-md5: 4260469cc245895d4a6ac93f0c0fca26
Homepage: https://git-scm.com/
Depends: libcurl, less
Filename: dists/stable/main/binary-arm/git_2.13.1_arm.deb
Size: 2273896
SHA256: e1f652397569aa724866b58febf3c66015f45e7980c949a6b07b59a30f562f73

$ apt list git
Listing... Done
git/stable,now 2.13.1 arm [installed]

# Operation on a pre-existing local repos
$ git status
error: sha1 mismatch 6716a1e830c184bae15d2cd5ea22ac30e13920ee
fatal: bad object HEAD
```

Thanks!
PS:  wondering, is there a way to downgrade to a previous package version ?

**Comment by Neo-Oli** [comment]
Happens to me too on aarch64.

Is this an upstream bug? Can anyone test git version 2.13.1 on a different distribution?

**Comment by ysalmon** [comment]
This is affecting me too and this is completely blocking.

**Comment by kwoodham** [comment]
Awe geez - wish I had come here first.  I just spent about a half an hour looking for ways to fix what I assumed to be a corrupted repository.  Running git 12.3.1.  Did a `packages update` and git has been downgraded to 12.3.0 - works again.

**Comment by fornwall** [comment]
This has now been fixed - the current git package is called `2.13.1.2` (and is really `2.13.1`, but numbered higher temporarily until the next git minor update upstream).
